lib/oli/delivery/sections.ex

```diff
@@ -1468,6 +1470,21 @@ defmodule Oli.Delivery.Sections do
     end)
   end
 
+  @spec maybe_update_contains_discusssions(Section.t()) :: Section.t()
+  def maybe_update_contains_discusssions(section) do
+    from(s in Section,
+      join: sr in assoc(s, :section_resources),
+      where: s.id == ^section.id,
+      where: fragment("?->>'status' = ?", sr.collab_space_config, "enabled"),
+      select: sr.id
```

Comment:
We should add a `limit: 1` here to ensure that this query does not return 500 items in the case when collab spaces are enabled on every page.

Reply:
Great idea! I'll do add the limit.